mappings/net/minecraft/entity/passive/BeeEntity.mapping

```diff
@@ -31,6 +31,14 @@ CLASS net/minecraft/class_4466 net/minecraft/entity/passive/BeeEntity
 		COMMENT A bee will start moving to a flower once this time in ticks has passed from a pollination.
 	FIELD field_30286 POLLINATION_FAIL_TICKS I
 		COMMENT The duration in ticks when a bee's pollination is considered failed.
+	FIELD field_30288 MAX_POLLINATED_CROPS I
+	FIELD field_30289 STING_POISON_DURATION_NORMAL_DIFFICULTY I
+	FIELD field_30290 STING_POISON_DURATION_HARD_DIFFICULTY I
```

Comment:
please make the difficulty prefix than suffix, according to yarn guidelines as these are durations than difficulties

Reply:
Should it be with or without `DIFFICULTY`?

Reply:
either, but usually we know peaceful easy hard are difficulties